doc/classes/Basis.xml

```diff
@@ -4,10 +4,12 @@
 		A 3×3 matrix for representing 3D rotation and scale.
 	</brief_description>
 	<description>
-		A 3×3 matrix used for representing 3D rotation and scale. Usually used as an orthogonal basis for a [Transform3D].
-		Contains 3 vector fields X, Y and Z as its columns, which are typically interpreted as the local basis vectors of a transformation. For such use, it is composed of a scaling and a rotation matrix, in that order (M = R.S).
-		Basis can also be accessed as an array of 3D vectors. These vectors are usually orthogonal to each other, but are not necessarily normalized (due to scaling).
+		The [Basis] built-in [Variant] type represents a 3x3 [url=https://en.wikipedia.org/wiki/Matrix_(mathematics)]matrix[/url] used for 3D rotation, scale, and shearing. It is frequently used within a [Transform3D].
+		A [Basis] is composed by 3 axis vectors, each representing a column of the matrix: [member x], [member y], and [member z]. The length of each axis ([method Vector3.length]) influences the basis's scale, while the direction of all axes influence the rotation. Usually, these axes are perpendicular to one another. However, when you rotate any axis individually, the basis becomes sheared. Applying a sheared basis to a 3D model will make the model appear distorted.
+		A [Basis] is [b]orthogonal[/b] if its axes are perpendicular from each other, which allows it to represent rotations. A basis is [b]normalized[/b] if the length of every axis is [code]1[/code]. A basis is [b]uniform[/b] if all axes share the same length (see [method get_scale]).
 		For a general introduction, see the [url=$DOCS_URL/tutorials/math/matrices_and_transforms.html]Matrices and transforms[/url] tutorial.
+		[b]Note:[/b] Godot uses the [b]right-hand coordinate system[/b] (+X is right, +Y is up, and +Z is back). This may be different from the coordinate system you are used to. For example, by convention, 3D assets are rotated towards the camera (+X is [b]left[/b], +Y is up, and +Z is [b]front[/b]). For more information, see the [url=$DOCS_URL/tutorials/assets_pipeline/importing_scenes.html#d-asset-direction-conventions]Importing 3D Scenes[/url] tutorial.
+		[b]Note:[/b] The basis matrices are stored in [url=https://www.mindcontrol.org/~hplus/graphics/matrix-layout.html]column-major[/url] order, which is the same as OpenGL. In comparison, DirectX uses row-major order.
```

Comment:
This is not quite accurate. The C++ code stores them in row-major. It would be more accurate to say they are *exposed* as column-major, not *stored* as column-major.